package.json

```diff
@@ -53,10 +53,12 @@
     "lite-server": "^2.2.0",
     "mocha": "3.2.0",
     "mocha-webpack": "0.7.0",
+    "moment": "2.0.0",
```

Comment:
Just curious are we using here only moment or also moment.js with locales ?  As with locales it is quite 'big' (70kb~). But I think this is something we should consider for  #7 when building it.